packages/forms/src/index.tsx

```diff
@@ -166,262 +269,470 @@ const useFieldRegister = <
         ref.current = element
       }
     },
+    onBlur,
+    onChange,
   }
 }
 
-// Context for keeping track of errors from the server
-interface FieldErrorContextProps {
+/**
+ * Context for keeping track of errors from the server.
+ */
+interface ServerErrorsContextProps {
   [key: string]: string
 }
-const FieldErrorContext = React.createContext({} as FieldErrorContextProps)
 
-const coerceValues = (
-  data: Record<string, string>,
-  coerce: (name: string, value: string) => any
-) => {
-  const coercedData: Record<string, any> = {}
-
-  Object.keys(data).forEach((name) => {
-    coercedData[name] = coerce(name, data[name])
-  })
-
-  return coercedData
-}
+const ServerErrorsContext = React.createContext({} as ServerErrorsContextProps)
 
-interface FormWithCoercionContext
-  extends Omit<React.HTMLProps<HTMLFormElement>, 'onSubmit'> {
+interface FormProps
+  extends Omit<React.ComponentPropsWithRef<'form'>, 'onSubmit'> {
   error?: any
   formMethods?: UseFormReturn
-  validation?: UseFormProps
+  config?: UseFormProps
```

Comment:
Here's the renamed prop

Reply:
👍 - might be an idea to add a jsdoc comment here, so they see hints on hover